src/Microsoft.ApiDesignGuidelines.Analyzers/UnitTests/EnumStorageShouldBeInt32Tests.Fixer.cs

```diff
@@ -29,5 +29,13 @@ protected override CodeFixProvider GetCSharpCodeFixProvider()
         {
             return new CSharpEnumStorageShouldBeInt32Fixer();
         }
+
+        #region CSharpUnitTests
+
+        #endregion
+
+        #region BasicUnitTests
+
+        #endregion
```

Comment:
My personal preference is to interleave the C# and VB tests so I don't have to look up and down the page to convince myself that every C# test has a matching VB test. (When I wrote that I was thinking of what I did when I was coding, but come to think of it, it makes it easier for me to review, as well, if each C# test is right next to its matching VB test.)